openstack/imageservice/v2/results.go

```diff
@@ -0,0 +1,349 @@
+package v2
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	
+	"github.com/rackspace/gophercloud"
+	//"github.com/rackspace/gophercloud/pagination"
+)
+
+// does not include the literal image data; just metadata.
+// returned by listing images, and by fetching a specific image.
+type Image struct {
+	Id string
+	
+	Name string
+	
+	Status ImageStatus
+	
+	Tags []string
+	
+	ContainerFormat *string
```

Comment:
I'll take care about it as well

On Tue, Aug 4, 2015 at 10:40 AM, jrperritt <notifications@github.com> wrote:

> In openstack/imageservice/v2/results.go
> <https://github.com/rackspace/gophercloud/pull/443#discussion_r36217679>:
>
> > +	"github.com/rackspace/gophercloud"
> > +	//"github.com/rackspace/gophercloud/pagination"
> > +)
> > +
> > +// does not include the literal image data; just metadata.
> > +// returned by listing images, and by fetching a specific image.
> > +type Image struct {
> > +	Id string
> > +	
> > +	Name string
> > +	
> > +	Status ImageStatus
> > +	
> > +	Tags []string
> > +	
> > +	ContainerFormat *string
>
> If the empty string is a valid option for ContainerFormat, then the
> *string is necessary. Otherwise, let's use string and only set that field
> if ContainerFormat is non-empty.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rackspace/gophercloud/pull/443/files#r36217679>.
>



-- 
With best regards
Ilya  Sviridov